homeassistant/util/unit_conversion.py

```diff
@@ -686,6 +688,20 @@ class UnitlessRatioConverter(BaseUnitConverter):
     }
 
 
+class ConcentrationConverter(BaseUnitConverter):
```

Comment:
Should we specify that this is a mass per volume concentration converter? There could be mass per mass and volume per volume concentrations too.

Reply:
Done 👍